.github/workflows/govulncheck.yml

```diff
@@ -18,16 +18,8 @@ jobs:
     steps:
       - uses: actions/setup-go@v5
         with:
-          go-version: "1.21"
+          go-version: "1.22"
           check-latest: true
       - uses: actions/checkout@v4
-      - uses: technote-space/get-diff-action@v6
```

Comment:
why remove?

Reply:
only useful when things are crazy slow, and they're no longer crazy slow.

Reply:
it really isn't needed now that ci is fast.